lightning/pkg/importer/table_import.go

```diff
@@ -1832,3 +1859,44 @@ func (tr *TableImporter) preDeduplicate(
 	)
 	return err
 }
+
+func estimateEngineDataSize(tblMeta *mydump.MDTableMeta, tblInfo *checkpoints.TidbTableInfo, isIndexEngine bool, logger log.Logger) int64 {
```

Comment:
Are these functions only used by the remote backend? If they are solely related to remote logic, it would be better to place them in the `remote` directory and the corresponding files for easier management.